File: app/src/main/java/com/tt/handsomeman/service/SignUpService.java
package com.tt.handsomeman.service;

import com.tt.handsomeman.response.StandardResponse;
import com.tt.handsomeman.util.Constants;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;
import retrofit2.http.Query;

public interface SignUpService {
    @FormUrlEncoded
    @POST(Constants.SIGN_UP_SUFFIX)
    Call<StandardResponse> doSignUp(@Query("type") String type, @Field("name") String name, @Field("mail") String mail, @Field("password") String password, @Field("rePassword") String rePassword);
}

File: app/src/main/java/com/tt/handsomeman/service/SignUpService.kt
package com.tt.handsomeman.service

import com.tt.handsomeman.response.StandardResponse
import com.tt.handsomeman.util.Constants

import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST
import retrofit2.http.Query

interface SignUpService {
    @FormUrlEncoded
    @POST(Constants.SIGN_UP_SUFFIX)
    fun doSignUp(@Query("type") type: String, @Field("name") name: String, @Field("mail") mail: String, @Field("password") password: String, @Field("rePassword") rePassword: String): Call<StandardResponse>
}
